src/dialogs/more-info/controls/more-info-input_timetable.ts

```diff
@@ -0,0 +1,192 @@
+import "@material/mwc-icon-button/mwc-icon-button";
+import "../../../components/ha-svg-icon";
+import "../../../components/ha-switch";
+import type { HaSwitch } from "../../../components/ha-switch";
+import "../../../components/paper-time-input";
+import { mdiDelete, mdiPlus } from "@mdi/js";
+import { HassEntity } from "home-assistant-js-websocket";
+import { BINARY_STATE_ON, BINARY_STATE_OFF } from "../../../common/const";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+
+declare type Event = {
+  time: string;
+  state: string;
+};
+
+@customElement("more-info-input_timetable")
+class MoreInfoInputTimetable extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({
+    attribute: false,
+    hasChanged(newVal: HassEntity, oldVal: HassEntity) {
+      if (!newVal) {
+        return false;
+      }
+      return (
+        !oldVal ||
+        JSON.stringify(oldVal.attributes.timetable) !==
+          JSON.stringify(newVal.attributes.timetable)
+      );
+    },
+  })
+  public stateObj?: HassEntity;
+
+  private _timetable: Array<Event> = [];
+
+  private _disable_reconfig = false;
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this.stateObj) {
+      return html``;
+    }
+    this._disable_reconfig = true;
+    return html`
+      ${this._timetable.map((event: Event, index: number) =>
+        this._renderEntry(event, index)
+      )}
+      <div class="add">
+        <mwc-icon-button
+          label="${this.hass!.localize(
+            "ui.dialogs.more_info_control.input_timetable.add"
+          )}"
+          @click=${() => {
+            this._addEntry();
+          }}
+        >
+          <ha-svg-icon .path=${mdiPlus}></ha-svg-icon>
+        </mwc-icon-button>
+        <p>
+          ${this.hass!.localize(
+            "ui.dialogs.more_info_control.input_timetable.add"
+          )}
+        </p>
+      </div>
+    `;
+  }
+
+  updated() {
+    this._disable_reconfig = false;
+  }
+
+  private _renderEntry(event: Event, index: number) {
+    const time = event.time.split(":");
+    return html` <div class="entry">
+      <paper-time-input
+        .hour=${time[0]}
+        .min=${time[1]}
+        format="24"
+        label=""
```

Comment:
```suggestion
```

Reply:
This is not a mistake.
Here is how it looks when removing the empty label:
![image](https://user-images.githubusercontent.com/19599059/114601080-c2fb9c00-9c9d-11eb-9a30-02c82563f587.png)

Reply:
Use `hide-label`

Reply:
Done